cmd/api-router.go

```diff
@@ -78,6 +78,45 @@ func getHost(r *http.Request) string {
 	return r.Host
 }
 
+func rejectNotImplemented(w http.ResponseWriter, r *http.Request) {
```

Comment:
you can just call it 
```suggestion
func notImplementedHandler(w http.ResponseWriter, r *http.Request) {
```